scripts/stac/imagery/tests/collection_test.py

```diff
@@ -156,3 +156,14 @@ def test_default_provider_missing() -> None:
         "providers"
     ]
     assert {"name": "Maxar", "roles": ["producer"]} in collection.stac["providers"]
+
+
+def test_capture_data_asset_present() -> None:
+    title = "Test Urban Imagery"
+    description = "Test Urban Imagery Description"
+    collection = ImageryCollection(title, description)
+
+    assert collection.stac["assets"]["capture_area"]["href"] == "./capture-area.geojson"
+    assert collection.stac["assets"]["capture_area"]["title"] == "Capture area"
+    assert collection.stac["assets"]["capture_area"]["type"] == "application/geo+json"
+    assert collection.stac["assets"]["capture_area"]["roles"] == ["metadata"]
```

Comment:
where do we add the checksums and filesize?

How big is this geojson do we need to compress it?

Reply:
Apparently we can use the `file` fields: https://github.com/stac-extensions/file#asset--link-object-fields

I've added an example from `CastleRockQuarry` dataset in the PR description. It's about 400bytes.